src/pages/signin/LoginForm/BaseLoginForm.js

```diff
@@ -270,7 +270,7 @@ function LoginForm(props) {
                     autoCorrect={false}
                     inputMode={CONST.INPUT_MODE.EMAIL}
                     errorText={formErrorText}
-                    hasError={hasError}
+                    hasError={hasServerError}
```

Comment:
So is this consistent with this convention?

> Adds red bottom border to inputs with form validation error (server error still results in a green border)

https://github.com/Expensify/App/pull/14627#issue-1560171649

Reply:
I don't think so – it was added in this PR: https://github.com/Expensify/App/pull/17168

I would say we should clarify the expected behaviour for the server-side errors because https://github.com/Expensify/App/pull/17168 and https://github.com/Expensify/App/pull/14627 seem to be conflicting in terms of the expected red border.

Reply:
Would you raise a discussion on Slack?

Reply:
https://expensify.slack.com/archives/C01GTK53T8Q/p1702465027956659

Reply:
So did we get a feedback to close this issue? 🙁

Reply:
Well, I think we can interpret this that we got a green light here